src/index.js

```diff
@@ -13,14 +13,24 @@ class DataImport extends React.Component {
     showSettings: PropTypes.bool,
   };
 
+  constructor(props) {
+    super(props);
+
```

Comment:
please remove empty lines

Reply:
I usually put an empty line after the `super`. So this one is not accidental.